xonsh/environ.py

```diff
@@ -101,11 +101,13 @@ def is_callable_default(x):
     """Checks if a value is a callable default."""
     return callable(x) and getattr(x, '_xonsh_callable_default', False)
 if ON_WINDOWS:
-    DEFAULT_PROMPT = ('{BOLD_INTENSE_GREEN}{user}@{hostname}{BOLD_INTENSE_CYAN} '
+    DEFAULT_PROMPT = ('{env_name}'
```

Comment:
Should this go before the user and host name? I don't know how it works elsewhere.